### PR TITLE
Update conn-faq-psn.md

### PR DESCRIPTION
**Pull request by NickStobbart**

### CODE REVIEW

File: articles/connectivity/conn-faq-psn.md [hunk 1]
```diff
@@ -141,9 +141,9 @@ Before additional IPv4 addresses can be allocated, you'll need to demonstrate yo
 
 PSN is a shared connection and all UKCloud PSN‑enabled customers use a shared bandwidth model.
 
-### Does each PSN environment have a maximum limit assigned to it?
+### Does each PSN connection have a maximum bandwidth limit assigned to it?
 
-We don't publish limits associated with PSN bandwidth as we capacity‑manage this link to ensure there is always enough bandwidth for our customers. Consumption of PSN bandwidth is based on the actual volume of data transferred (outbound) per month.
+We do not publish limits associated with PSN bandwidth as we capacity manage the link sto ensure there is always enough bandwidth for our customers. Consumption of PSN bandwidth is based on the actual volume of data transferred (outbound) per month.
```
Comment (SteveHallUKCLoud) [inline articles/connectivity/conn-faq-psn.md:146]: typo - the link sto ensure

**Review by SteveHallUKCLoud — CHANGES_REQUESTED** [review]
Also, a lot of the changes seem to clash with brand guidelines (ie: "don't" changed to "do not")

File: articles/connectivity/conn-faq-psn.md (hunk 1 above)
Comment (suehighmoor) [reply articles/connectivity/conn-faq-psn.md]: Fixed typo and reverted back to using contractions per KC style guidelines

**Review by SteveHallUKCLoud — APPROVED** [review]
LGTM

**Review by suehighmoor — APPROVED** [review]
LGTM

**Pull request merged**